content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/print_on_validation.rst

```diff
@@ -0,0 +1,156 @@
+=======================
+Printable delivery PDFs
+=======================
+
+Automatically print delivery-related PDFs documents and labels in Odoo, containing package recipient
+details, contents, or handling instructions.
+
+The following PDFs can be configured to print upon validating an *Inventory* operation (e.g.
+receipt, picking, delivery orders, quality checks):
+
+#. Delivery slip
+#. Return slip
+#. Product labels of items in the order
+#. Lot and serial number labels
```

Comment:
Suggestion to add internal references to each section, since you mentioned "jump to the related section" 😉
```suggestion
#. :ref:`Delivery slip <inventory/shipping_receiving/delivery-slip>`
#. :ref:`Return slip <inventory/shipping_receiving/return-slip>`
#. :ref:`Product labels of items in the order <inventory/shipping_receiving/product-labels>`
#. :ref:`Lot and serial number labels <inventory/shipping_receiving/lot-sn-labels>`
```